ch32v307/template.yaml

```
@@ -0,0 +1,49 @@
+name: algorithm-test
+variants:
+  - name: algorithm-test
```

```suggestion
name: ch32v307
variants:
  - name: CH32V307
```

Although this is not the final algorithm description, better use the chip name for heading.

---

Changed it